mantle/kola/tests/coretest/core.go

```diff
@@ -179,31 +163,6 @@ func TestPortSsh() error {
 	return nil
 }
 
-func TestDockerEcho() error {
-	//t.Parallel()
-	return util.RunCmdTimeout(DockerTimeout, "docker", "run", "busybox", "echo")
-}
-
-func TestDockerPing() error {
-	//t.Parallel()
-	return util.RunCmdTimeout(DockerTimeout, "docker", "run", "busybox", "ping", "-c4", "coreos.com")
-}
```

Comment:
I wonder if we specifically have a `docker` test or two in the external tests.. if not it might be good to add at least one that does these two things.

Reply:
No no. We should definitely delete them here. I'm just wondering if we have any coverage for this at all running right now (i.e. external tests).

Reply:
We have the [docker.basic](https://github.com/coreos/fedora-coreos-config/blob/b6280b2ade4d7520a9477d12d9989a59190f5db4/tests/kola/docker/basic) test in FCOS that tests running a basic container with docker works. It doesn't specifically test whether that container has internet access, so I agree it would be good to add an external test to cover it.